src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -1,15 +1,79 @@
 ---
-title: Alerts and applied intelligence transition guide 2022
+title: Alerts and applied intelligence transition guide
 tags:
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - EOL
   - Transition guide
 metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
 
+## What's changed [#changed]
+
+Starting in late 2021, we've reorganized the navigation UI for alerts and applied intelligence. Although the alerts and applied intelligence navigation has changed pretty dramatically, all the features you use are still there.
+
+We're slowly rolling out this update to all of our users.
+
+![A screenshot of the new alerts and applied intelligence left navigation.](./images/alerts-ai-categories.png "The new alerts and applied intelligence left navigation")
+
+<figcaption>The new UI includes these new categories.</figcaption>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="nav-changes"
+    title="An overview of alerts and applied intelligence nav changes"
+  >
+
+  ## Analyze [#analyze]
+
+The [overview page](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page) shows your most recent open, closed, and muted violations.
+
+The **Issues & activity** page shows your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## Detect [#detect]
+
+The [alert conditions (policies) page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) lists the policies where your conditions are created and organized. Create new policies here.
+
+The [anomaly detection page](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/) lists your expanded anomaly detection configurations. You can add or edit the configurations here.
+
+## Correlate [#correlate]
+
+The [sources page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources) shows your configured incident intelligence sources. You can add new sources or configure existing ones.
```

Comment:
maybe reword as "shows the sources you select to configure"... what? "incident intelligence" sounds like what New Relic would call it, not what I as a user would call it. If nothing else, what kinds of sources can I select? Not the whole laundry list but maybe a few standard things and/or some things our competitors cannot configure alerting on?

Reply:
Totally agree. I've tried to expand on this without making it a laundry list :)